### PR TITLE
May 15th

### PR DESCRIPTION
**Pull request by tlandryIntact**
Component section, first pass

### CODE REVIEW

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc [hunk 1]
```diff
@@ -4,33 +4,40 @@
 
 
 === Intact Financial Corporation
+Wildfire risk in Canada is prominent and even though major events do not occur every year, they can cause unprecedented damage. Costs from the wildfire events of summer 2021 in British Columbia reached $77 and $78 millions in insured damage at White Rock Lake and Lytton, respectively. Wildfire activity is expected to go up due to an increase in fire-prone conditions across the country. It is therefore essential that insurers build capacities to monitor wildfire risk and occurrence to protect customers and manage risk exposure.
 
-- Component: 
-  * Wildfire hazard map
+IFC's participation to this pilot was conducted by the Centre for Climate and Geospatial Analytics (CCGA), a team of experts operating at Intact Data Lab. The main focus of this participation is to better understand end-to-end hazard and risk modelling workflows, in turn supporting the climate services required for decision-making in the business. This participation is also intended to further open up Intact Lab to the outside world, by providing information on wildfire risks and climate resiliency in the context of the insurance industry.
```
Comment (jongad) [inline 2023-07_Pilot-ER/sections/04-03-dri.adoc]: Is the IFC abbreviation detailed previously?

Comment (jongad) [inline 2023-07_Pilot-ER/sections/04-03-dri.adoc]: We use "outside world" a lot internally, but perhaps we want to use something different here.

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc [hunk 2]
```diff
@@ -4,33 +4,40 @@
...
+
+==== About Us
+Intact is the largest provider of Property & Casualty (P&C) insurance in Canada. Its purpose is to help people, businesses and society prosper in good times and be resilient in bad times. The company has been on the front lines of climate change with our customers for more than a decade – getting them back on track and helping them adapt. As extreme weather is going to get worse over the next decade, Intact intends to double down on adapting to this changing environment and be better prepared for floods, wildfire and extreme heat. 
+
+With close to 500 data, AI, machine learning and pricing experts, the Intact Data Lab has deployed almost 300 AI models in production to-date, helping transform outperformance advantage. It is focused on improving risk selection and making  operations as efficient as possible while creating outstanding interactions
```
Comment (jongad) [inline 2023-07_Pilot-ER/sections/04-03-dri.adoc]:
```suggestion
With close to 500 experts in data, AI, machine learning and pricing, the Intact Data Lab has deployed almost 300 AI models in production to-date, helping transform outperformance advantage. It is focused on improving risk selection and making operations as efficient as possible while creating outstanding interactions 
```

Comment (jongad) [inline 2023-07_Pilot-ER/sections/04-03-dri.adoc]: Not sure what you mean by "helping transform outperformance advantage".

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc [hunk 3]
```diff
@@ -4,33 +4,40 @@
...
+
+==== About Us
+Intact is the largest provider of Property & Casualty (P&C) insurance in Canada. Its purpose is to help people, businesses and society prosper in good times and be resilient in bad times. The company has been on the front lines of climate change with our customers for more than a decade – getting them back on track and helping them adapt. As extreme weather is going to get worse over the next decade, Intact intends to double down on adapting to this changing environment and be better prepared for floods, wildfire and extreme heat. 
+
+With close to 500 data, AI, machine learning and pricing experts, the Intact Data Lab has deployed almost 300 AI models in production to-date, helping transform outperformance advantage. It is focused on improving risk selection and making  operations as efficient as possible while creating outstanding interactions 
+with customers. Within Intact’s Data Lab, the Centre for Climate and Geospatial Analytics (CCGA) uses weather, climate, and geospatial data along with machine learning models to develop risk maps and other specialized products to the business.
```
Comment (jongad) [inline 2023-07_Pilot-ER/sections/04-03-dri.adoc]:
```suggestion
with customers. Within Intact’s Data Lab, the Centre for Climate and Geospatial Analytics (CCGA) uses weather, climate, and geospatial data along with machine learning models and claims data to develop risk maps and other specialized products to the business.
```

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc [hunk 4]
```diff
@@ -4,33 +4,40 @@
...
+
+==== Scope of Work
+The project centered the efforts around these challenges:
+
+ * Identify current usages of wildfire risk maps at Intact by interviewing various business units;
```
Comment (jongad) [inline 2023-07_Pilot-ER/sections/04-03-dri.adoc]:
```suggestion
 * Identify current usages of wildfire maps at Intact by interviewing various business units;
```

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc [hunk 5]
```diff
@@ -4,33 +4,40 @@
...
+ * Inform internal architectural, infrastructure and procurement processes of new geospatial standards and trends;
+ * Identify and develop insurance wildfires risk use cases, to build Disaster and Climate Resilience.
+
+These activities should, when possible, align with the best practices and standards of the OGC, as well as current and proposed themes in OGC's DWG.
+
+==== Approach
+To develop its component, Intact migrated its previous proprietary wildfire hazard model to a new private on-premise data science environment. For all key inputs to the model, external connections to several open data repositories were established. To facilitate these access tests, early in the project, several public open source - climate model outputs, Earth observation, weather, geospatial - datasets were vetted by the appropriate cybersecurity boards. The tests informed experts of changes in platforms offerings, of new data products specifications and of current authoritative scientific references. The following datasets were accessed specifically to produce an updated wildfire hazard map (GeoTiff), and to compute regional risk indicator by postal codes (CSV):
```
Comment (jongad) [inline 2023-07_Pilot-ER/sections/04-03-dri.adoc]:
```suggestion
To develop its component, Intact migrated its previous proprietary wildfire hazard model to a new private on-premise data science environment. For all key inputs to the model, external connections to several open data repositories were established. To facilitate these access tests, several public open source datasets, such as climate model outputs, Earth observations, weather and geospatial, were vetted by the appropriate cybersecurity boards. The tests informed experts of changes in platforms offerings, of new data products specifications and of current authoritative scientific references. The following datasets were accessed specifically to produce an updated wildfire hazard map (GeoTiff), and to compute regional risk indicator by postal codes (CSV):
```

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc [hunk 6]
```diff
@@ -4,33 +4,40 @@
...
   * Fire Weather Index (FWI) daily maps
   * Land cover maps
   * Drought conditions
   * Digital Elevation Model (DEM)
   * Population density
 
-- Outputs:
-  * Wildfire hazard map, in GeoTiff
-  * Regional risk indicators, as tabular output in CSV format
+In parallel, CCGA polled IFC's stakeholders about current and potential uses of wildfire risk products within their operations. This information was used to identify use cases supporting this pilot project, as well as prospective proof-of-concepts for wildfire resiliency. Additionally, the team adopted a multi-hazard approach, seeking to apply learnings obtained from flood risk mapping, benchmarking and decision making. For instance, CCGA routinely manipulate exposure data representing insured risks throughout the country, as well as claims received for previous catastrophic events.
```
Comment (jongad) [inline 2023-07_Pilot-ER/sections/04-03-dri.adoc]: The term "multi-hazard" seems to suggest that we're developing a combined wildfire/flood model.

File: 2023-07_Pilot-ER/sections/07-lessons_learned.adoc [hunk 7]
```diff
@@ -31,11 +31,7 @@ The various organizations and institutes that contribute to the Climate Resilien
 
 - At present participants have only implemented the first Drought Index (SPI) using precipitation data from the Copernicus Climate Data Store (CDS), but are open to / need other data sources.
 
-- One of the objectives of the pilot is also to lower the barriers for users accessing CDS/ADS (Atmosperic Data Store) data and services and engage with a broader user community. Knowing from users which are these barriers (gaps) will allow this pilot to evolve as well.
-
-- A universal, well-defined, climate service workflow (from row data to information) as a roadmap to guide developers/users through the process.
-
-- Improve Sentinel-2 data cube performance, add Climate data, add vegetation fuel type classification, add a wildfire risk assessment workflow.
+- Definition of common documentation guidelines for terms and licences of endpoints would facilitate approval in cybersecurity-heavy environments. Individual security reviews for each participants and their endpoints is a tedious process. Having common terms as defined in OGC's master agreementx, alongside applicable open source licences, would facilitate endpoint whitelisting.
```
Comment (jongad) [inline 2023-07_Pilot-ER/sections/07-lessons_learned.adoc]:
```suggestion
- Definition of common documentation guidelines for terms and licences of endpoints would facilitate approval in cybersecurity-heavy environments. Individual security reviews for each participants and their endpoints is a tedious process. Having common terms as defined in OGC's master agreement, alongside applicable open source licences, would facilitate endpoint whitelisting.
```

**Review by jongad — CHANGES_REQUESTED** [review]

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc (hunk 1 above)
Comment (tlandryIntact) [reply 2023-07_Pilot-ER/sections/04-03-dri.adoc]: Good catch, but here I think it's fine to use the expression "externally". Always easier to memorize to explain.

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc (hunk 2 above)
Comment (tlandryIntact) [reply 2023-07_Pilot-ER/sections/04-03-dri.adoc]: Corporate lingo taken straight from annual report. I might just cut the sentence.

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc (hunk 1 above)
Comment (tlandryIntact) [reply 2023-07_Pilot-ER/sections/04-03-dri.adoc]: Applied change

File: 2023-07_Pilot-ER/sections/04-03-dri.adoc (hunk 6 above)
Comment (tlandryIntact) [reply 2023-07_Pilot-ER/sections/04-03-dri.adoc]: Adjusted wording

**Comment by tlandryIntact** [comment]
Merging first PR to fork

**Pull request merged**